lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -178,7 +180,7 @@ def jsonld?
         end
 
         def performance_data?
-          @performance_data == true
+          @performance_data == true && !jsonld?
         end
```

Comment:
Since performance data is not displayed when returning json-ld, move the check here to avoid doing any performance calculations when not needed.